editor/export/editor_export_platform_apple_embedded.cpp

```diff
@@ -2785,10 +2785,10 @@ void EditorExportPlatformAppleEmbedded::_initialize(const char *p_platform_logo_
 	Ref<Image> img = memnew(Image);
 	const bool upsample = !Math::is_equal_approx(Math::round(EDSCALE), EDSCALE);
 
-	ImageLoaderSVG::create_image_from_string(img, p_platform_logo_svg, EDSCALE, upsample, false);
+	ImageLoaderSVG::create_image_from_string(img, p_platform_logo_svg, EDSCALE, upsample, HashMap<Color, Color>{});
```

Comment:
This one's more of a nitpick, but the more common implementation is `A()` over `A{}`. The latter has a niche if using `std::initializer_list`, as it shortens `A({ 1, 2, 3 }) to `A{ 1, 2, 3 }`, but no arguments make this an entirely stylistic choice. Go with whichever you prefer
```suggestion
	ImageLoaderSVG::create_image_from_string(img, p_platform_logo_svg, EDSCALE, upsample, HashMap<Color, Color>());
```